elastic/integrations
# [Airflow] Add Airflow Dashboard

- Enhancement


## What does this PR do?

This PR adds overview dashboard for Airflow.

## Checklist




## How to test this PR locally

Setup and configure airflow to send metrics to statsd server. 
Add integration with set host and port.
Metrics should start coming and confirm in dashboard


## Related issues



- Closes #5001 


## Screenshots

<img width="1704" alt="Screenshot 2023-02-06 at 9 23 08 AM" src="https://user-images.githubusercontent.com/102972658/216923645-83526ede-874b-4320-8a2c-bf9d9a1b4763.png">
